### PR TITLE
Tst return none inplace series

### PR DESCRIPTION
**Pull request by r-toroxel**
verify we return None for all inplace calls in /series

related: https://github.com/pandas-dev/pandas/pull/35181#event-3531612625

### CODE REVIEW

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 1]
```diff
@@ -22,8 +22,10 @@ def test_drop_duplicates(any_numpy_dtype, keep, expected):
     tm.assert_series_equal(tc.duplicated(keep=keep), expected)
     tm.assert_series_equal(tc.drop_duplicates(keep=keep), tc[~expected])
     sc = tc.copy()
-    sc.drop_duplicates(keep=keep, inplace=True)
+    return_value = return_value = sc.drop_duplicates(keep=keep, inplace=True)
+    assert return_value is None
     tm.assert_series_equal(sc, tc[~expected])
+    assert return_value is None
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:28]: extra here

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 2]
```diff
@@ -40,8 +42,10 @@ def test_drop_duplicates_bool(keep, expected):
...
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:48]: extra

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 3]
```diff
@@ -84,22 +88,27 @@ def test_drop_duplicates_categorical_non_bool(self, dtype, ordered):
         tm.assert_series_equal(tc1.duplicated(), expected)
         tm.assert_series_equal(tc1.drop_duplicates(), tc1[~expected])
         sc = tc1.copy()
-        sc.drop_duplicates(inplace=True)
+        return_value = return_value = sc.drop_duplicates(inplace=True)
+        assert return_value is None
         tm.assert_series_equal(sc, tc1[~expected])
+        assert return_value is None
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:94]: extra

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 4]
```diff
@@ -113,22 +122,28 @@ def test_drop_duplicates_categorical_non_bool(self, dtype, ordered):
         tm.assert_series_equal(tc2.duplicated(), expected)
         tm.assert_series_equal(tc2.drop_duplicates(), tc2[~expected])
         sc = tc2.copy()
-        sc.drop_duplicates(inplace=True)
+        return_value = return_value = sc.drop_duplicates(inplace=True)
+        assert return_value is None
         tm.assert_series_equal(sc, tc2[~expected])
+        assert return_value is None
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:128]: extra

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 5]
```diff
@@ -113,22 +122,28 @@ def test_drop_duplicates_categorical_non_bool(self, dtype, ordered):
...
 
         expected = Series([False, True, True, False, True, True, False])
         tm.assert_series_equal(tc2.duplicated(keep=False), expected)
         tm.assert_series_equal(tc2.drop_duplicates(keep=False), tc2[~expected])
         sc = tc2.copy()
-        sc.drop_duplicates(keep=False, inplace=True)
+        return_value = return_value = sc.drop_duplicates(keep=False, inplace=True)
+        assert return_value is None
         tm.assert_series_equal(sc, tc2[~expected])
+        assert return_value is None
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:146]: extra

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 6]
```diff
@@ -141,19 +156,25 @@ def test_drop_duplicates_categorical_bool(self, ordered):
...
 
         expected = Series([True, True, False, False])
         tm.assert_series_equal(tc.duplicated(keep="last"), expected)
         tm.assert_series_equal(tc.drop_duplicates(keep="last"), tc[~expected])
         sc = tc.copy()
-        sc.drop_duplicates(keep="last", inplace=True)
+        return_value = return_value = sc.drop_duplicates(keep="last", inplace=True)
+        assert return_value is None
         tm.assert_series_equal(sc, tc[~expected])
+        assert return_value is None
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:171]: extra

File: pandas/tests/series/methods/test_drop_duplicates.py [hunk 7]
```diff
@@ -141,19 +156,25 @@ def test_drop_duplicates_categorical_bool(self, ordered):
...
 
         expected = Series([True, True, True, True])
         tm.assert_series_equal(tc.duplicated(keep=False), expected)
         tm.assert_series_equal(tc.drop_duplicates(keep=False), tc[~expected])
         sc = tc.copy()
-        sc.drop_duplicates(keep=False, inplace=True)
+        return_value = return_value = sc.drop_duplicates(keep=False, inplace=True)
+        assert return_value is None
         tm.assert_series_equal(sc, tc[~expected])
+        assert return_value is None
```
Comment (jreback) [inline pandas/tests/series/methods/test_drop_duplicates.py:180]: extra

File: pandas/tests/series/methods/test_sort_values.py [hunk 8]
```diff
@@ -81,7 +82,8 @@ def test_sort_values(self, datetime_series):
             "you must create a copy"
         )
         with pytest.raises(ValueError, match=msg):
-            s.sort_values(inplace=True)
+            return_value = s.sort_values(inplace=True)
```
Comment (jreback) [inline pandas/tests/series/methods/test_sort_values.py:85]: this likely doesn't execute, can you revert (and if you are testing withing other raises tests)

**Review by jreback — CHANGES_REQUESTED** [review]
some comments, ping on green

**Comment by r-toroxel** [comment]
Thx @jreback. Resolved.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @r-toroxel

**Pull request merged**